Jenkinsfile

```diff
@@ -314,7 +314,7 @@ try {
 
   stage('Package') {
     def parallelSteps = [:]
-    def osBuilds = isStableBuild() ? ['centos7', 'centos8'] : ['centos7']
+    def osBuilds = isStableBuild() ? ['centos7', 'centos8'] : ['centos7', 'centos8']
```

Comment:
```suggestion
    def osBuilds = ['centos7', 'centos8']
```

Reply:
we'll revert this juste before merge, but we let it for testing purpose